appengine/flexible/Pubsub/Pubsub.Sample/Controllers/HomeController.cs

```diff
@@ -63,7 +63,7 @@ public IActionResult Index(MessageForm messageForm)
                     Data = ByteString.CopyFromUtf8(messageForm.Message)
                 };
                 pubsubMessage.Attributes["token"] = _options.VerificationToken;
-                _publisher.PublishAsync(pubsubMessage);
+                _publisher.PublishAsync(pubsubMessage).Wait();

```

Comment:
```suggestion
                await _publisher.PublishAsync(pubsubMessage);

```

And then the method signature needs to change to:

```
public async Task<IActionResult> IndexAsync(...)
```

Reply:
Done.